conu/apidefs/backend.py

```diff
@@ -0,0 +1,53 @@
+"""
+Definition for a backend class
+"""
+
+import logging
+
+from .container import Container
+from .image import Image
+
+
+__version__ = "0.0.1-alpha"
+
+
+def set_logging(level=logging.INFO):
```

Comment:
I would prefer to create a class with several static classes.

Reply:
I don't understand, can you provide an example?

Reply:
Like this https://github.com/rebase-helper/rebase-helper/blob/master/rebasehelper/logger.py

Reply:
I can see that for rebase helper it works nicely. For conu, it's just too complicated -- I doubt this project will ever have more than one (at most two) loggers -- therefore I think that having a single function to initiate logging is simpler for current use case. Once we are in a need to have multiple loggers using different configuration, we can revisit.